File: Data Structures & Algorithms/pow-x-n/submission-2.py
class Solution:
    def myPow(self, x: float, n: int) -> float:
        answer = x
        y = x
        if n==0:
            return 1
        if (n>=0):
            for i in range(0, n-1):
                answer *=y
        else:
            n *= -1
            for i in range(0, n-1):
                answer *=y

            answer = 1/answer
        return float(answer)
        

File: Data Structures & Algorithms/string-matching-in-an-array/submission-2.cpp
class Solution {
public:
    vector<string> stringMatching(vector<string>& words) {
        unordered_set<string> answer;

        for(int i = 0; i < words.size(); i++){
            for(int j = 0; j < words.size(); j++){
                if(i != j and words[j].find(words[i]) != string::npos){
                    answer.insert(words[i]);
                }
            }
        }
        return vector<string>(answer.begin(), answer.end());
    }
};